platforms/std/docs/STD-installation.md

```diff
@@ -296,7 +296,7 @@ using it!
 
 3. Install Splice Machine:
 
-   Unpack the tarball `gz` file that you downloaded: <a href="https://s3.amazonaws.com/splice-releases/2.5.0.1802/standalone/splicemachine-2.5.0.1729.tar.gz">https://s3.amazonaws.com/splice-releases/2.5.0.1802/standalone/splicemachine-2.5.0.1729.tar.gz</a>
+   Unpack the tarball `gz` file that you downloaded: <a href="https://s3.amazonaws.com/splice-releases/2.7.0.1835/standalone/SPLICEMACHINE-2.7.0.1835.standalone.tar.gz">https://s3.amazonaws.com/splice-releases/2.7.0.1835/standalone/SPLICEMACHINE-2.7.0.1835.standalone.tar.gz</a>
```

Comment:
@bklo94 Our master version is 2.8.0 instead of 2.7.0.

Reply:
however, the 2.8 branch has not been released? I think Brandon's point on this readme change was to point to the currently 'released' standalone, which is 2.7.0 not 2.5.0?